assets/js/script.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+
+const titleId = document.getElementById('title');
+const authorId = document.getElementById('author');
+const bookContainer = document.getElementById('book_container');
+const addButton = document.getElementById('form_btn');
+const formBook = document.getElementById('book_form');
+
+let library = [];
+
+function displayBook(library) {
+  const data = library.map((book) => `
+        <div>
+            <p>${book.title}</p> 
+            <p>${book.author}</p>
+            <button id="${book.id}" onclick="removeElement(this)" class="delete">Remove</button>
+        </div><hr>`);
+  bookContainer.innerHTML = data.join(' ');
+}
+
+formBook.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const id = Math.floor(Math.random() * 10000);
+  const newBook = new Book(id, this.title.value, this.author.value);
+  library = [...library, newBook];
+  addStorage(library);
+  displayBook(library);
+  this.title.value = '';
+  this.author.value = '';
+});
+
+function Book(id, title, author) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+}
+
+/* Add Storage */
+
+function addStorage(library) {
+  localStorage.setItem('newBook', JSON.stringify(library));
+}
+
+/* Remove Storage */
+function removeElement(element) {
+  library = library.filter((i) => i.id !== element.id);
+  addStorage(library);
+  displayBook(library);
+}
```

Comment:
- [ ] The implementation for removing a book is not functioning as expected. It's expected to remove/delete a book by its ID and only a book that's remove button clicked should be removed from the book list. Currently, after adding books, when I click on any of the books remove buttons, all the boos are removed from the storage. Kindly work on only a single book by its ID removed and not the entire list. 👍